consensus/src/round_manager.rs

```diff
@@ -621,11 +621,17 @@ impl RoundManager {
     /// 2) call process_certificates(), which will start a new round in return.
     async fn process_vote(&mut self, vote: &Vote) -> anyhow::Result<()> {
         let round = vote.vote_data().proposed().round();
-        debug!(
+
+        info!(
             self.new_log(LogEvent::ReceiveVote)
                 .remote_peer(vote.author()),
-            "{}", vote
+            vote = %vote,
+            vote_epoch = vote.vote_data().proposed().epoch(),
+            vote_round = vote.vote_data().proposed().round(),
+            vote_id = vote.vote_data().proposed().id(),
+            vote_state = vote.vote_data().proposed().executed_state_id(),
```

Comment:
So these are just so you can query by them?  Otherwise it's in the `vote` field already right?

Reply:
yep! although some of these fields are truncated in the `vote` field (vote_id and vote_state) and here we don't truncate them (on purpose)